chapters/th/_toctree.yml

```diff
@@ -0,0 +1,173 @@
+- title: 0. Setup
```

Comment:
We only need to keep the entries for the translated chapters. In this case, please remove everything except the parts for chapter 3

Reply:
Got it! sorry for mistakes here. Fixed accordingly.